src/Cart/index.js

```diff
@@ -1,15 +1,32 @@
-import React from 'react'
-import './style.css'
+import React, { Component } from 'react';
+import './style.css';
 
-function Cart() {
+class Cart extends Component {
+
+  state = {
+    flag: false,
+  }
+
+  render() {
   return (
-    <div className='cart'>
+    <div className={this.state.flag === true ? 'cart cart_is_red' : 'cart cart_is_green'}>
```

Comment:
Why to duplicate `cart` style as well? You can use such formula instead.
```
<div className={'cart '+ this.state.flag === true ? 'cart_is_red' : 'cart_is_green'}>
```
Or. You can refer to wonderful package that handles conditional styling, classnames:
https://www.npmjs.com/package/classnames